src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/repositories/MetadataStoreRepository.kt

```diff
@@ -9,8 +10,13 @@ import java.sql.ResultSet
 import java.sql.Timestamp
 
 @Repository
-class MetadataStoreRepository(private val connection: Connection,
-                              private val table: String) {
+class MetadataStoreRepository(
+    private val connection: Connection,
+    @Qualifier("table") private val table: String,
+    @Qualifier("queryLimit") private val queryLimit: String,
+    @Qualifier("trimReconciledScale") private val trimReconciledScale: String,
```

Comment:
Can these 2 trim variables be passed to the TrimService which in turn passes them to the metadatastore deleteRecords method?

Reply:
Yeh can do that

Reply:
Thanks.